cluster/ci/config/prow/config.yaml

```diff
@@ -1469,3 +1469,13 @@ periodics:
     - name: token
       secret:
         secretName: openshift-bot-token
+- name: ci-kubernetes-conformance-node-e2e-rhel
+  agent: jenkins
+  labels:
+      master: ci.openshift.redhat.com
+  interval: 12h
```

Comment:
What about each 2 hours? Or is it to often?

Reply:
That was the original schedule, right? So let's use 2h

Reply:
Yes sir

Reply:
My bad.  Long time no crontab.

Fixed.